style.css

```diff
@@ -362,6 +408,9 @@ section .title::after{
     top: 0;
     background: #2653d9;
 }
+.skills-content .right .dsl::before{

```

Comment:
line python

basically whatever the content is, line is after that name.. It's not random